app-domain-common/model/src/main/java/org/codeforvictory/recipes/domain/common/model/meal/Meal.kt

```diff
@@ -0,0 +1,3 @@
+package org.codeforvictory.recipes.domain.common.model.meal
+
+class Meal(val id: String, val name: String, val imageThumbUrl: String)
```

Comment:
this might be a stupid question. Not data class cause of native?

Reply:
I almost asked the same.  I'm assuming it's just a matter of don't make it data class unless you need the freebies it gives?

Reply:
This was a slip! Thank you both for catching it up. Addressed: [`4606ac9` (#13)](https://github.com/CodeForVictory/recipes-mvi-playground/pull/13/commits/4606ac9ce1e6d7155cba6ed2f1f3145ed6ee3929)